server/src-lib/Hasura/Eventing/ScheduledTrigger.hs

```diff
@@ -192,26 +192,26 @@ data ScheduledEventFull
   , sefRetryConf     :: !STRetryConf
   , sefHeaders       :: ![EventHeaderInfo]
   , sefComment       :: !(Maybe Text)
+  , sefCreatedAt     :: !UTCTime
```

Comment:
Let's not send `createdAt` for cron events as it is an implementation detail and can be confusing to consumers.